package.json

```diff
@@ -24,20 +24,20 @@
     "react-load-script": "^0.0.6",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
-    "react-static": "^5.1.12",
+    "react-static": "^5.6.8",
```

Comment:
not a big deal, just curious why the upgrade?

Reply:
Buried in my commit message https://github.com/cityofaustin/janis/pull/210/commits/f4c147da7ede208e803036e266faf76396de5dc4

> This solves a build problem I was running into, though I'm not entirely sure why. It seems like it's a `TileGroup` or `TileGroupSet` issue, but I'm not ready to track it down.
> 
> The old version would fail the build but the new version shows the following warning
> 
```
Check the top-level render call using <div>. See https://fb.me/react-warning-keys for more information.
    in TileGroup
    in TileGroupSet
    in div
    in div
    in Theme
...
```